src/main/kotlin/com/ekino/oss/jcv/idea_plugin/language/JcvLexerAdapter.kt

```diff
@@ -0,0 +1,8 @@
+package com.ekino.oss.jcv.idea_plugin.language
+
+import com.intellij.lexer.FlexAdapter
+
+class JcvLexerAdapter : FlexAdapter(JcvLexer(null))
+
+
+private val value = listOf("val1","value",)
```

Comment:
Is there a use to this line ? It looks weird 😛

Reply:
Wow good catch!